### ISSUE TITLE
Notary Application：Pluskit

### ISSUE THREAD

**Issue by PluskitOfficial** [filecoin-project/notary-governance#448]
# Notary Application

To apply to be a Fil+ Notary, please review the Notary Overview [here](https://github.com/filecoin-project/notary-governance/tree/main/notaries#overview) and then fill out the following form. 

## Core Information
- Name: Pluskit
- Affiliated organization:
- Website / Social Media:
-  On-chain address to be notarized (recommend using a new address): f1tgnlhtcmhwipfm7thsftxhn5k52velyjlazpvka
- Region of Operation: Asia minus GCN
- Use case(s) to be supported: General
- DataCap Requested: 1 PiB

_Please respond to the questions below in pargraph form, replacing the text saying "Please answer here". Include as much detail as you can in your answer!_

### Time Commitment
Describe the nature and duration of your affiliation with the Filecoin project. Please include relevant Github handles, miner ids, significant projects or contributions (with links).
```
Pluskit is a Fil+ ecosystem tool platform product, dedicated to providing participants with tools with practical functions, strong operability and rich types to promote activity and boost prosperity of the whole ecosystem.

Pluskit team was established in June 2021, and the product was officially launched in October 2021 to comprehensively present the DataCap data of general verification to help Protocol Labs and community members intuitively understand the acquisition and allocation of DataCap, thereby effectively avoid“private transactions”.

In December 2021, Pluskit team was invited to participate in the Notary Governance Call to present a project showcase, which won a large number of likes from community members;

In January 2022, Pluskit received Next Step Microgrant;

In February 2022, Pluskit was included in the Filecoin Community Roadmap.

Github: https://github.com/PluskitOfficial/Pluskit
```

### Stake Exposure
Please cite total token at stake (currently available, locked as collateral, vesting over time) and any substantiating evidence.
```
None.
```

### In-protocol Reputation
Please describe (in detail) your activity and tenure as a member of the Filecoin community. Please note (with links where possible) any contributions made to implementations of Filecoin, the spec, documentation, or to substantially help the Filecoin ecosystem grow. 
```
Pluskit team was established in June 2021, and the product was officially launched in October 2021 to comprehensively present the DataCap data of general verification to help Protocol Labs and community members intuitively understand the acquisition and allocation of DataCap, thereby effectively avoid“private transactions”.

Currently, we are developing DataCap data of Large Datasets verification and Automatic verification to help community users more comprehensively understand the current status of real data storage implementation.

In December 2021, Pluskit was invited to participate in the Notary Governance Call to present a project showcase. It's also a great honor for Pluskit to be guided by the PL and the Filecoin Foundation for several times.
```

### In-protocol Security
Please describe your contributions to the security of Filecoin and the duration over which you've made contributions. Please also include any links or references that can substantiate your contributions. 
```
None.
```

### Organizational Reputation
Please describe the nature of your organization, including the country of registration, size of the organization, and time since inception.
```
Pluskit team was established in June 2021. The team consists of 5 freelancers with many years of experience in the Filecoin industry. Pluskit is developed mainly for overseas markets and serves the entire ecosystem.

Since Pluskit was launched, it has received close attention and recognition from Protocol Labs and the Filecoin Foundation. In December 2021, it was invited to participate in the Notary Governance Call to present a project showcase and was recognized by notaries. Since then, it has been actively contacted and recognized by a large number of community members. 
```

Please share any relevant details to help substantiate information about your organization (website, named officers, links to social media profiles).
```
Website：http://pluskit.io/
Twitter：@PluskitOfficial
Slack：@PluskitOfficial
```

Please share any relevant external information regarding your organization (e.g. news articles, social media profiles, etc.)
```
None.
```

### Individual Reputation
Please share links to at least 2 of your (personal) social media profiles (or accounts that you are able to use) and the approximate size of your audience (i.e., followers, subscribers) for each one.
```
Twitter：@PluskitOfficial
```

### Use Case Diversity
(Optional) Any additional information you'd like to share about the use case(s) you plan to support?
```
We will accept any use case, but will be more inclined to support developer tools and Web 3.0 applications.
```

# Allocation Plan
### Allocation Strategy
How do you plan on allocating the DataCap requested above? Please describe your allocation strategy with as much specificity as you can. This includes the target amount per client and rate at which you'll allocate DataCap.
```
We believe that the entire industry needs to allow more real data to be stored. For this purpose, we will try our best to relax the application restrictions on the basis of strict screening with real storage purposes.
1. There will be no restriction on the region, and any regional application request will be accepted.
2. There will be no restriction on the type of subject, and applications from any subject such as enterprises, organizations, and individuals are accepted.
3. We will not limit the type of data content, and will accept data of any data type on the basis of compliance with the law.

At the same time, in order to avoid private transactions, we will have strict requirements for data storage:
1. For the same application subject, the storage amount of the same node is not more than 30% of the amount of DataCap acquired in the same batch.
2. When the remaining amount of the applicant's DataCap is greater than 25% of the total amount of data (including applications from other notaries), new applications will not be accepted.
3. Only full public data or non-public data subject to random checks are accepted.
4. We will only accept those who will agree to give feedback with storage records (for the first time, only commitments are made, and subsequent actual feedback should be used as the basis for judgment)

When reviewing the application, we will evaluate and allocate from six aspects, including applicant's basic information, data information, industry reputation, time commitment, usage plan, and historical records.
1. Basic information: name, time of birth/registration, nationality/region of registration (certificate documents are required, licenses provided by enterprises/organizations, and ID cards provided by individuals)
2. Data information: industry, data content, total amount of data to be stored, whether it can be disclosed (whether it supports spot checks if it is not disclosed)
3. Industry reputation: industry status, honor level, social media followers
4. Time commitment: Contribution time to their industry
5. Usage plan: storage node ratio, whether to feed back storage records
6. History record: DataCap acquisition source, acquisition amount, DataCap remaining amount

We plan to allocate at least 50% of DataCaps within half a year, and the amount of DataCap applied for by each clients does not exceed 20% of our holdings, and each client can be allocated at most once a week.
```

How do you plan on securing the DataCap to ensure your organization (and its delegated members) are the ones allocating the DataCap?
```
We plan to store private keys in hard wallets.
```

### Client Due Diligence
How will you vet the clients that are applying for DataCap? What questions will you ask to ensure your trust is placed well and that clients can properly handle the DataCap you intend to allocate to them? 
```
1. We will evaluate clients from various aspects, such as qualifications, industry honors, etc., to ensure that people with bad intentions are filtered to obtain DataCap to the greatest extent possible before allocation.
2. We will confirm the usage plan in the questioning session, and ask for feedback on the actual usage after obtaining the DataCap, and will also continue to follow up the actual usage.
3. Our initial allocation will be relatively small. If the client is responsible for using DataCap, we will gradually increase allocation; if it is used improperly, we will suspend the subsequent vetting and allocation for this client.

We plan to ask the following questions:
1.Are you an individual or a business/organization? If an individual, please specify the name, date of birth, and nationality; if a company/organization, please specify the company name, registration time, and registration area. At the same time, please provide the corresponding supporting documents (individuals provide ID cards, enterprises/organizations provide licenses, which can be sent by E-mail).
2.What is your industry? What data will be stored? How much do you plan to store? Can the data be made public (if not, are we allowed to conduct random checks at any time?)
3.Please share information and relevant supporting documents or links that illustrate your industry status.
4.Did you get any honors? Please provide relevant supporting documents or links.
5.Please provide your social media account and indicate the number of followers.
6.Please describe the duration of your involvement in your industry and give a brief description of your industry.
7.Please specify the storage location, miner ID and storage proportion of your data to be stored. We require any data to be stored on the same node no more than 30% of the same batch of DataCap acquisitions.
8.If you get a DataCap, do you agree to send the storage record back here within 15 days of storage?
9.Did you get a DataCap? If yes, please attach all application links and inform the current remaining amount.
```

What processes will you employ when granting additional DataCap to a client that has previously been verified? This includes confirming that the client is not improperly using the DataCap they were previously granted, i.e., making deals with a single SP entity.
```
We will clearly require in advance that the DataCap used by the same clients on the same node shall not more than 30% of the amount acquired in the same batch.
We will require clients to announce storage details in a timely manner and check the storage proportion before additional allocations.
Our initial allocation will be relatively small. If the client is responsible for using DataCap, we will gradually increase it; if it is used improperly, we will suspend subsequent vetting and allocation for the client
```

### Bookkeeping Plan
Do you plan on conducting all your allocation decisions in public (e.g. Github repo), private (e.g. over email, Telegram, etc), or both? 
```
Yes, our allocation decisions will be made public on Github.
```

Where do you plan on keeping a publicly accessible record of all your allocation decisions?
```
All allocation decisions will be kept/updated both on Github and locally.
```

### Engagement in Program
How much time per week, on average, are you willing to dedicate to participating in the Fil+ program? This includes making DataCap allocations (direct and/or Large Datasets), comments on discussion/issues, attendance in governance calls, messages in Slack, etc.
```
For general verification, we plan to conduct uniform vetting and allocation from 16:00 to 18:00 every Tuesday and Friday, and the processing time will be 4 hours per week.

In addition, we will spend an hour a day on Slack and actively monitor and process a large share of applications, which will be 5 hours in total.

We also plan to attend Notary Governance Call at least once a month.

To sum up, we will spend an average of 9 hours per week on Fil+ projects.
```

### Past allocation 
Have you previously received DataCap to allocate before? If so, please link to any previous applications.
```
None.
```

Cumulatively, how much DataCap have you previously successfully allocated? 
```
None.
```

**Comment by galen-mcandrew** [comment]
[Pluskit - Round 3 Notary Election Scorecard.pdf](https://github.com/filecoin-project/notary-governance/files/8320879/Pluskit.-.Round.3.Notary.Election.Scorecard.pdf)

**Comment by PluskitOfficial** [comment]
Dear @galen-mcandrew , thanks for scoring. We would like to add two parts to our application.
1. Allocation Strategy
As for the vetting process, we will take action according to the following principles and plans:
Timeliness: Reply within 1 working day after receiving the application.
Compatibility: We will give opportunity for clients to modify their information, and review within 1 working day.
Rigour: If clients do not reply within 7 working days, the application will be closed;
Please check out our scoring sheet:
<img width="971" alt="1" src="https://user-images.githubusercontent.com/94815279/159647256-1f7f1da5-2d2c-42a4-84bf-0d67697453a0.png">

2. Bookkeeping Plan
In addition to disclose all allocation decisions to the community, our product itself has been working to make allocation records public.
When you click on the home page of Pluskit (pluskit.io), you will see the application information of all the applicants, as well as the processing status of the notary public, including the processing time, cycle, results, etc. These information is completely open to the public.
<img width="677" alt="2" src="https://user-images.githubusercontent.com/94815279/159647287-af77630d-7753-4c9c-bb86-d693dec6a26a.png">
<img width="676" alt="3" src="https://user-images.githubusercontent.com/94815279/159647299-5fda64e4-df77-4cf2-aa55-f8a3a4cd3634.png">
<img width="403" alt="4" src="https://user-images.githubusercontent.com/94815279/159647305-ccb69da3-a701-4ac8-9903-2728133288af.png">

If there is any dispute about the allocation, community members can also send a message to us directly via Github/Slack(both@PluskitOfficial). We will promptly deal with it and contact the clients for confirmation when necessary. We will disclose the information collected during the vetting process and the allocation basis. 

At the same time, we are constantly optimizing our allocation strategy to make it easier for clients to join the Filecoin Network while following community rules.

**Comment by Kevin-FF-USA** [comment]
Hi  @PluskitOfficial **based on this Notary election cycle's final scoring, you/your organization has qualified to be a Fil+ Notary!** You will be receiving your final scored rubric soon, along with the total allocation of Datacap based on rubric scoring.

In order to confirm your participation as a Notary in the Fil+ ecosystem, please respond to the following:

1.  Please confirm that the region of operation for client applications you will focus on is Asian minus GCN.  

2. Please confirm each of the following items below (you can do this by quoting each of the following bullets and adding a line under each section agreeing that you'll abide by these operational principles).  

	•	Upfront Disclosures: Prior to being confirmed as a Notary, Notaries are expected to disclose all relevant addresses which they control, have a financial stake in, or are strongly connected to by other means. For the disclosure, the Notary should state the relevant addresses and the nature of the relationship .
	
	•	Promoting Client Best Practices: Notaries agree to educate approved clients about the best practices for using their DataCap (e.g. how to request additional services from miners, storing data redundantly across many miners, etc). Some reference information can be found here. 
	
	•	Commitment to efficiently serving the Network: Notaries agree to serve as fiduciaries of the Network, striving to work towards bringing useful data onto Filecoin and improving the experience for clients to do so. Notaries should generally be able to respond to Client applications and updates within 3 days, and should be comfortable communicating with Clients and Notaries if an application needs to be redirected. 
	
	•	No Self Dealing: To prevent conflicts of interest, Notaries should not allocate DataCap to Clients over which they control the private keys, or to a Client who intends to specifically spend the allocated DataCap with an address affiliated with the Notary. When in doubt, Notaries should bias towards transparency (i.e. public disclosure) or to getting a different Notary to handle the individual request. 
	
	•	Operating in Good Faith: Notaries hold a position of trust in the network, and as such it is expected that they operate keeping the Principles of this mechanism in mind. While each form of abuse cannot be exhaustively defined, Notaries are expected to bias towards caution and act in a way that promotes transparency. Notaries should expect to potentially receive requests or questions for allocation decisions (within reason) - and should make decisions with this in mind. 
	
	•	Community Governance Participation: It is expected that you will participate in the program 9 hours a week. Along with data allocation, participation in Github issues and Slack conversations, Notaries are to make an effort to regularly attend the scheduled Governance calls. As these calls are a forum to shape this process, it is important to ensure Notaries are present to provide their context, with discussions and input. 
	
3. Please list any addresses you are affiliated with, and state the nature of the relationship. Please refer to the first bullet point in (2) for the definition of "affiliated", and bias towards transparency when in doubt.  

4. Please affirm that you will abide by the allocation / client due diligence plan you laid out above.  

5. (If ready) Please confirm the address that should receive DataCap. This is the address which you will use to sign messages on-chain to verify clients (through using a Ledger and the Fil+ Registry App). If you have an active (non-zero) DataCap grant from a previous election cycle, please provide a different address here. 
	•	If you do not have a ledger already, we recommend ordering directly from the Manufacturer for the Nano S or Nano X (link: https://shop.ledger.com/pages/ledger-nano-x)

**Comment by galen-mcandrew** [comment]
[Pluskit - UPDATED - Round 3 Notary Election Scorecard.pdf](https://github.com/filecoin-project/notary-governance/files/8373126/Pluskit.-.UPDATED.-.Round.3.Notary.Election.Scorecard.pdf)

**Comment by PluskitOfficial** [comment]
Hi, @Kevin-FF-USA Thanks a lot!

1.We confirm that we will focus on Asia minus GCN.

2.We confirm all items below:

**Comment by PluskitOfficial** [comment]
Hi, @Kevin-FF-USA Thanks a lot!

1.We confirm that we will focus on Asia minus GCN.

2.We confirm all items below:

> Upfront Disclosures: Prior to being confirmed as a Notary, Notaries are expected to disclose all relevant addresses which they control, have a financial stake in, or are strongly connected to by other means. For the disclosure, the Notary should state the relevant addresses and the nature of the relationship

Confirmed.

> Promoting Client Best Practices: Notaries agree to educate approved clients about the best practices for using their DataCap (e.g. how to request additional services from miners, storing data redundantly across many miners, etc). Some reference information can be found here.

Confirmed.

> Commitment to efficiently serving the Network: Notaries agree to serve as fiduciaries of the Network, striving to work towards bringing useful data onto Filecoin and improving the experience for clients to do so. Notaries should generally be able to respond to Client applications and updates within 3 days, and should be comfortable communicating with Clients and Notaries if an application needs to be redirected.

Confirmed.

> No Self Dealing: To prevent conflicts of interest, Notaries should not allocate DataCap to Clients over which they control the private keys, or to a Client who intends to specifically spend the allocated DataCap with an address affiliated with the Notary. When in doubt, Notaries should bias towards transparency (i.e. public disclosure) or to getting a different Notary to handle the individual request.

Confirmed.

> Operating in Good Faith: Notaries hold a position of trust in the network, and as such it is expected that they operate keeping the Principles of this mechanism in mind. While each form of abuse cannot be exhaustively defined, Notaries are expected to bias towards caution and act in a way that promotes transparency. Notaries should expect to potentially receive requests or questions for allocation decisions (within reason) - and should make decisions with this in mind.

Confirmed.

> Community Governance Participation: It is expected that Notaries make an effort to regularly attend the scheduled Governance calls. As these calls are a forum to shape this process, it is important to ensure Notaries are present to provide their context, learnings, and input.

Confirmed.

3.We don't have any addresses.

4.We will abide by the allocation / client due diligence plan we laid out above.

5.We confirm that our address is:   f1tgnlhtcmhwipfm7thsftxhn5k52velyjlazpvka

**Comment by galen-mcandrew** [comment]
@PluskitOfficial 

Please fill out this form to move forwards with the ratification process: https://airtable.com/shrs55Lzbm1wJTIRw

**Comment by PluskitOfficial** [comment]
@galen-mcandrew Received and Thanks!   We sumbitted just now~

**Comment by Kevin-FF-USA** [comment]
Hello @PluskitOfficial!

We have one Pluskit account on file for Slack: PluskitOfficial (https://filecoinproject.slack.com/team/U02JBHJFCKG). This is the account we will be communicating with. If there are any other Slack handles you wish to be included, please comment their handles below.

**Comment by PluskitOfficial** [comment]
Hi, @Kevin-FF-USA~
Just to confirm no other Slack handles needs to be included.

**Comment by PluskitOfficial** [comment]
## Notary Ledger Verified
#### Message sent to Filecoin Network
> message CID: bafy2bzaceb3safxkfh4du5r6u3vygerppw6xentlshkyjc5vmdoeemuiqdtje
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceb3safxkfh4du5r6u3vygerppw6xentlshkyjc5vmdoeemuiqdtje

**Comment by galen-mcandrew** [comment]
## Request Approved
#### Address 
> f1tgnlhtcmhwipfm7thsftxhn5k52velyjlazpvka
#### Datacap Allocated
> 1PiB

**Comment by PluskitOfficial** [comment]
## Notary Ledger Verified
#### Message sent to Filecoin Network
> message CID: bafy2bzacedakcd7kmrz7dgqjdtvwyw4kmaiejriujpwnybcafgxv6dzqnfvhw
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacedakcd7kmrz7dgqjdtvwyw4kmaiejriujpwnybcafgxv6dzqnfvhw

**Comment by galen-mcandrew** [comment]
## Request Approved
#### Address 
> f1tgnlhtcmhwipfm7thsftxhn5k52velyjlazpvka
#### Datacap Allocated
> 0PiB

**Comment by galen-mcandrew** [comment]
bafy2bzaceahlfk4agw64ezlozlxnhdc3tzry3lt4e5pttkrpojmjv35zxx2tg
